README.md

```diff
@@ -355,6 +355,8 @@ K9s uses aliases to navigate most K8s resources.
 | To view and switch to another Kubernetes context (Pod view)                     | `:`ctx⏎                       |                                                                        |
 | To view and switch directly to another Kubernetes context (Last used view)      | `:`ctx context-name⏎          |                                                                        |
 | To view and switch to another Kubernetes namespace                              | `:`ns⏎                        |                                                                        |
+| To switch between the last active view/command (like how "cd -" works)          | `-`                           | Navigation that adds breadcrumbs to the bottom are not commands        |
+| To go back through previous views/commands                                      | `b`                           | Same as above                                                          |
```

Comment:
I think 'b' is already in use in some views aka to run benchmarks ;(
Perhaps we can use '-' as the last 2 cmd toggle as you have here but then useuse '<' and '>' to nav the command history?